src/module/Event/src/Event/EventManager.php

```diff
@@ -243,13 +250,14 @@ public function logEvent(
         }
 
         $this->getObjectManager()->persist($log);
+        $this->getEventManager()->trigger('log', $this, ['log' => $log]);
```

Comment:
Also open question: currently, notifications are tied to events. We can leave it like that (and create new event types for external incoming notifications like chat / commenting system) or split up notifications from events (e.g. by persisting an arbitrary payload in the notification that contains everything a notification wants).

Reply:
okay for now